### DISCUSSION TITLE
export Dependency track report with 1.6 CycloneDX version

### DISCUSSION THREAD

**Discussion by MIO1100** [DependencyTrack/dependency-track#4241] [Q&A]
Hello. I want to use some integrations for your tool, and i need to export CycloneDX with 1.6 version. Now I use 4.10.1 version. Do you have a plan to grow 1.5 version to 1.6? Or can i configurate this choice? Have you already done this in the new versions? I didn't find it.

**Comment by fupgang** [comment]
Hi,

> Or can i configurate this choice?

Exporting SBOMs is currently hard coded with version 1.5.

https://github.com/DependencyTrack/dependency-track/blob/5235dd869afb3150da557642f821bb85de4e41fe/src/main/java/org/dependencytrack/parser/cyclonedx/CycloneDXExporter.java#L96-L104

Right away, I didn't find an issue to reflect the above TODO. Pherhaps you should create one.
